### PR TITLE
GE + Duration, Ordinal, TimeGrain, Time

### PR DESCRIPTION
**Pull request by desmart18**
Summary: 

Added initial support for ka_GE for

- Duration
- Ordinal
- TimeGrain
- Time

### CODE REVIEW

**Comment by desmart18** [comment]
Hi @patapizza 

when I was building Georgian AmountOfMoney, a problem occurred with symbol of Lari (₾) (Georgian currency). while trying to build, it says: lexical error in string/character literal at character '\8382'. 

I tried doing this, but it doesn't work: https://stackoverflow.com/questions/44882118/haskelllexical-error-in-string-character-literal-at-character-d

I'm using stack version 1.7.1 on Ubuntu 16.04

Can you please advice me how to solve this problem?

File: Duckling/AmountOfMoney/KA/Rules.hs [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+--     [ regex "თეთრი?ს?(ად)?თ?"
+--     ]
+--   , prod = \_ -> Just . Token AmountOfMoney $ currencyOnly TETRI
+--   }
+
+ruleLari :: Rule
+ruleLari = Rule
+  { name = "Lari"
+  , pattern =
+    [ regex "ლარი?ს?(ად)?თ?| ?GEL ?"
```
Comment (patapizza) [inline Duckling/AmountOfMoney/KA/Rules.hs]: try:
```
[ regex "(ლარი?ს?(ად)?თ?)|GEL"
```

BTW `GEL` should be part of the common currency symbol rule in `AmountOfMoney/Rules.hs`.

**Comment by patapizza** [comment]
Hi @desmart18, thank you for the awesome contribution!

Can you keep `AmountOfMoney` as a separate pull request? This will speed up the review process.
I've added an inline comment with a suggestion.

File: Duckling/AmountOfMoney/KA/Rules.hs (hunk 1 above)
Comment (desmart18) [reply Duckling/AmountOfMoney/KA/Rules.hs]: Thanks for your comment, I took it into account and tried to do it, however, when I try to use Georgian symbol for Lari (₾) following error occurs: 

/home/david/duckling4/Duckling/AmountOfMoney/Rules.hs:64:9: error:
    lexical error in string/character literal at character '\8382'

**Comment by haoxuany** [comment]
Hi @desmart18 ,

Thank you for your contribution! We deeply appreciate it. I'd like you to know that we're currently working on reviewing this diff internally. Since this is a very large diff, we expect that it will take quite a bit longer than usual.

In the meantime, I'd appreciate it if there are any more updates to the pull requests, that you put it in either a separate pull request, or notify me ahead of time so that I can do something about it. For some reason, our internal tools are not very good are doing internal and external merges at the same time.

**Comment by desmart18** [comment]
Hi @haoxuany,

Thanks for reply! I would like you to know that we have Georgian Amount-Of-Money ready right now, but we are waiting and not making a new pull request. 

Also, I couldn't solve the issue with the symbol of Georgian Amount of money - ₾, if I use this symbol, project is not building and following error occurs: lexical error in string/character literal at character '\8382'.

I have tried many solutions, but none worked. The last thing I tried was using \x8382 and I was able to build it, but it couldn't recognize the symbol ₾ again. So, is there any way you could help us with it?

thanks a lot

**Comment by patapizza** [comment]
@desmart18 Feel free to submit a new pull request with `AmountOfMoney`, based on master :) We'll work on getting the Georgian currency symbol in.

**Comment by desmart18** [comment]
Hi @patapizza 

My version of the project that I try to commit successfully builds and passes all tests on my computer, on Linux 16.04, stack 1.7.1, CABALVER=1.24 GHCVER=8.0.2, but on travis on the same configuration:"os": "linux", "env": "CABALVER=1.24 GHCVER=8.0.2", an error occurs, that says:

"No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated"

Our version was 1 commit behind of the project and this problem occurred after the merge.

I don't get the reason, so can you please advice me how to solve it?

**Comment by haoxuany** [comment]
Hi @desmart18,

Sorry for the delay. Your pull request has finally landed! 😄 Thank you for your contribution!

**Pull request closed without merging**